e2e/promise-async-handling/__tests__/unhandledRejectionTest.test.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+'use strict';
+
+const {promisify} = require('util');
+
+test('w/o event loop turn after rejection', () => {
+  Promise.reject(new Error('REJECTED'));
+});
```

Comment:
Failures of this test made me update other tests in this PR. I have to force event loop to make a single turn to catch this as unhandled promise rejection.